frontend/src/app/patients/ArchivePersonModal.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { gql, useMutation } from "@apollo/client";
+import Modal from "react-modal";
+import Button from "../commonComponents/Button";
+import { displayFullName, showNotification } from "../utils";
+import "./ArchivePersonModal.scss";
+
+import Alert from "../commonComponents/Alert";
+import { toast } from "react-toastify";
+
+const ARCHIVE_PERSON_RECORD = gql`
+  mutation ArchivePerson($id: String!, $deleted: Boolean!) {
+    setPatientIsDeleted(id: $id, deleted: $deleted) {
+      internalId
+    }
+  }
+`;
+
+interface Props {
+  person: any;
```

Comment:
```suggestion
  person: Patient;
```

Looks like this is called from `ManagePatients.tsx` where the `Patient` interface is defined